srcpkgs/python3-gitlab/template

```diff
@@ -1,21 +1,18 @@
 # Template file for 'python3-gitlab'
 pkgname=python3-gitlab
-version=1.12.1
-revision=5
+version=3.6.0
+revision=1
 wrksrc=python-gitlab-$version
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-requests python3-six"
+depends="python3-setuptools python3-requests python3-requests-toolbelt python3-six"
```

Comment:
why is setuptools required? six is not mentioned at all in upstream.

Reply:
they were already there so i didn't consider removing them. now that i have tested without them, the package works just fine.